.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and run tests
+on: [push, pull_request]
+env:
+  app_id: inputstream.ffmpegdirect
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: "Debian package test"
+          os: ubuntu-20.04
```

Comment:
ubuntu-latest is better choice, with this we will not need to update this once 20.04 runner will be EOLed.

Reply:
I need ffmpeg 4 to be able to build this. So I can't use latest. But something is not working anyway as its using ffmpeg5 for some reason.